bitnami/mongodb/values.yaml

```diff
@@ -89,6 +89,18 @@ auth:
   ##
   # existingSecret: name-of-existing-secret
 
+tls:
+ ## Enable or disable MongoDB TLS Support
+  enabled: true
```

Comment:
Disable this by default to avoid breaking changes

Reply:
disabled